burn-import/src/burn/node/conv2d.rs

```diff
@@ -154,6 +154,8 @@ mod tests {
             Conv2dConfig::new([3, 3], [3, 3]).with_padding(PaddingConfig2d::Valid),
         ));
 
+        graph.register_input_output(vec!["input".to_string()], vec!["output".to_string()]);
```

Comment:
I'm not sure to understand why we need to add this part to all nodes. Why don't we use the actual name of each input and output instead of hardcoding the names ?

Reply:
Now the graph requires input and output information. Previously we were using the first node's inputs and last node's outputs as graph inputs and outputs but this is not true. Some middle node could also start needed inputs. I had to fix each test to add the input and output names.